attendance-portal/src/Components/Details.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
We use React 17.0, we don't need to import React here. Please use jsxImportSource
```suggestion
/** @jsxImportSource @emotion/core */
```